validation/config.go

```diff
@@ -35,6 +39,10 @@ func init() {
 	if err := env.Parse(&defaultConfig); err != nil {
 		panic(err)
 	}
+
+	if defaultConfig.MaxGroupMemPrice < minDefaultMaxGroupMemPrice {
+		panic("invalid max group memory price")
+	}
```

Comment:
let's not do this.  i agree with the sentiment but we should handle it more gracefully.  probably need to refactor the way this validation works - it was added in a rush to make a (2018) testnet more user-friendly.

just skip it for now

Reply:
`fmt.Println` instead of panic?

Reply:
nah.

Reply:
everything still works fine with a lower value, right?  it's just that the prices are lower than we'd like.

Reply:
kk. makes sense